### ISSUE TITLE
Upgrade to 4.3.0 markdown editor encoding issues

### ISSUE THREAD

**Issue by jimbacha** [Shazwazza/Articulate#359]
Upgrading from 4.2.1 to 4.3.0 breaks the markdown editor on the front-end. In the preview looks fine but on the front-end renders html break tags, spans and such.

**Comment by Shazwazza** [comment]
Hi, what is the property editor set to for the Articulate Markdown Editor Data Type? There's some release notes for 4.3.0 here https://github.com/Shazwazza/Articulate/releases/tag/v4.3.0 including some upgrade information.  Let me know if that solves your issue, else if you can provide steps to replicate since I'm using this actively on my blog and it works.

I wonder if part of the issue might be if you have HTML snippets within your markdown? I ask because for the new Markdig converter things need to be html encoded (see https://github.com/Shazwazza/Articulate/blob/a3e792e2de8dfb6508e479d96a6055420dbc86c9/src/Articulate/PropertyEditors/ArticulateMarkdownPropertyEditor.cs#L40)

**Comment by Shazwazza** [comment]
Hi, I found the issue. I was using a Markdig extension for Prism highlighting but it turns out that this extension is not needed and actually causes the code to not be encoded thus having to forcefully encode it which is not what is required. Turns out Markdig automatically adds the language/class attributes automatically so prism will 'just work' without the extension (which also means that the code is html encoded correctly).

I'll push an update shortly.
